compiler-rt/lib/builtins/cpu_model/riscv.c

```diff
@@ -369,6 +366,9 @@ void __init_riscv_feature_bits(void *PlatformArgs) {
     return;
 
   initRISCVFeature(Hwprobes);
+
+  __riscv_feature_bits.length = RISCV_FEATURE_BITS_LENGTH;
+  __riscv_vendor_feature_bits.length = RISCV_VENDOR_FEATURE_BITS_LENGTH;
```

Comment:
This would work, but it might be more obvious if this was moved into `initRISCVFeature`, maybe where the other fields in the structure are being set?